plugins/auth/index.js

```diff
@@ -70,14 +71,15 @@ exports.register = (server, options, next) => {
     });
 
     /**
-     * Generates a jwt that is signed and has a 12h lifespan
+     * Generates a jwt that is signed and has a 2h lifespan
      * @method generateToken
      * @param  {Object} profile Object from generateProfile
      * @return {String}         Signed jwt that includes that profile
      */
     server.expose('generateToken', profile => jwt.sign(profile, pluginOptions.jwtPrivateKey, {
         algorithm: ALGORITHM,
-        expiresIn: EXPIRES_IN
+        expiresIn: EXPIRES_IN,
+        jwtid: guid()
```

Comment:
Since we are simply generating a random uuid each time, we aren't really using the features that aguid was built to perform. 

We should probably just use [uuid](https://www.npmjs.com/package/uuid) instead, as that is a self-contained implementation that does everything we want.

Reply:
I don't really have a preference for either module.
Updating